hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/state/codec/VirtualBlobKeySerializer.java

```diff
@@ -29,12 +27,12 @@ public class VirtualBlobKeySerializer implements KeySerializer<VirtualBlobKey> {
     static final long DATA_VERSION = 1;
 
     @Override
-    public int deserializeKeySize(ByteBuffer byteBuffer) {
-        return VirtualBlobKey.sizeInBytes();
+    public int getSerializedSize() {
+        return 0;
```

Comment:
Should be `VirtualBlobKey.sizeInBytes()`?

Reply:
It should. Fixed